tasks/system_probe.py

```diff
@@ -1143,6 +1149,52 @@ def tempdir():
         shutil.rmtree(dirpath)
 
 
+def kitchen_prepare_btfs(ctx, files_dir, arch=CURRENT_ARCH):
+    sudo = "sudo" if not is_root() else ""
+    btf_dir = "/opt/datadog-agent/embedded/share/system-probe/ebpf/co-re/btf"
+
+    if arch == "x64":
+        arch = "amd64"
+
+    download_btfs(ctx, btf_dir, arch)
+
+    try:
+        co_re_programs = glob.glob("/opt/datadog-agent/embedded/share/system-probe/ebpf/co-re/*.o")
+        generate_minimized_btfs(
+            ctx,
+            source_dir=f"{btf_dir}/btfs-{arch}",
+            output_dir=f"{btf_dir}/minimized-btfs",
+            input_bpf_programs=co_re_programs
+        )
+
+        ctx.run(f"cd {btf_dir}/minimized-btfs && " +
+                f"{sudo} tar -cJf minimized-btfs.tar.xz * && " +
+                f"{sudo} mv minimized-btfs.tar.xz {files_dir}")
+    except:
```

Comment:
This is too wide of a net, and is catching other errors besides the `bpftool` version. I'd suggest calling `check_for_bpftool_compatability(ctx)` in a `try/except` block, on its own.